### DISCUSSION TITLE
Aggregation algorithm

### DISCUSSION THREAD

**Discussion by omnyaatef22** [securefederatedai/openfederatedlearning#376] [Q&A] [answered]
can I use the Robust algorithm for the aggregation? and what are the aggregation algorithms supported by openFL ?

**Comment by alexey-gruzdev** [comment, marked as answer]
Hi @omnyaatef22 ! Thanks for the interest in OpenFL!:) 

In OpenFL, currently we have few algorithms for robust aggregation: 

- [FedCurv](https://github.com/intel/openfl/tree/develop/openfl-tutorials/interactive_api/PyTorch_Histology_FedCurv) 
- Adaptive Aggregation [Adagrad/Adam/Yogi](https://openfl.readthedocs.io/en/latest/overriding_agg_fn.html#command-line-interface)
- Extended custom: you can define you own aggregation algorithm via [extension mechanism](https://openfl.readthedocs.io/en/latest/overriding_agg_fn.html#example-of-a-custom-aggregation-function)

**Reply by alexey-gruzdev** [reply]
@omnyaatef22 We've added corresponding section in our [README](https://github.com/intel/openfl#supported-aggregation-algorithms) . If the question is solved, please marked this as resolved.
